src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -494,7 +518,7 @@ public void addNotification(int accountId, int chatId, int msgId) {
                             lines = new ArrayList<>();
                             accountInbox.put(chatId, lines);
                         }
-                        lines.add(line);
+                        lines.add(tickerLine);
```

Comment:
```suggestion
                        lines.add(shortLine);
```

Reply:
see discussion above

Reply:
maybe better discuss on point to the code here:

- it is true, that this was the variable `line` but not `tickerLine` before
- however, in the old code, `line` was  converted to a _ticker-line-including-sender_, see L406 in the old code - and `tickerLine` was never a _ticker-line-including-sender_, even though that was probably the idea

this is indeed confusing, took me a while to re-understand this after these weeks of writing the PR